config.py

```diff
@@ -43,3 +43,9 @@
 
 # Email default domains
 USER_EMAIL_DEFAULT_DOMAINS = ["@gmail.com", "@apple.com"]
+
+# Secret key for session cookies
+SECRET_KEY = "coffeebuddy"
```

Comment:
What is this needed for? It does not seem to be used in the code.

Reply:
`SECRET_KEY` is used by flask internally for encrypting application cookies.
https://flask.palletsprojects.com/en/2.2.x/config/#SECRET_KEY

In real application this must be of course a real secret.

Reply:
Ok, understood